qiskit_ibm_provider/ibm_backend.py

```diff
@@ -339,6 +340,58 @@ def target_history(self, datetime: Optional[python_datetime] = None) -> Target:
         self._convert_to_target()
         return self._target
 
+    def _check_circuit(
+        self,
+        circuit: Union[QuantumCircuit, Schedule, List[Union[QuantumCircuit, Schedule]]],
+    ) -> None:
+        """Check if circuit contains operations on faulty qubits or gates.
+
+        Args:
+            circuit: transpiled circuit to check.
+            backend: backend the circuit will run on.
+
+        Raises:
+            IBMInputValueError: if circuit contains operations on faulty qubits or gates.
+        """
+        self.properties()
+        faulty_qubits = self._properties.faulty_qubits()
+        faulty_gates = self._properties.faulty_gates()
+        faulty_edges = [
+            tuple(gate.qubits) for gate in faulty_gates if len(gate.qubits) > 1
+        ]
+
+        if isinstance(circuit, List):
```

Comment:
Should we check for `List` here or in `backend.run`? Also would this work if a `schedule` is passed in?